xml/admin_upgrade.xml

```diff
@@ -194,19 +194,19 @@
      <para>
       Create the LDAP related pillars:
      </para>
-<screen>&prompt.bash;<command>entrypoint.sh</command> <option>bundle exec rails runner \
-    'Velum::LDAP.ldap_pillar_settings!({}).each \
-    {|key, value| Pillar.create(pillar: Pillar.all_pillars[key.to_sym], \
-    value: value)}'</option></screen>
+<screen>&prompt.bash;<command>entrypoint.sh bundle exec rails runner \
+'Velum::LDAP.ldap_pillar_settings!({}).each \
+{|key, value| Pillar.create(pillar: Pillar.all_pillars[key.to_sym], \
+value: value)}'</command></screen>
```

Comment:
This was indented for a reason. It should be clear that the command continue in the following line(s).